### PR TITLE
feat(mobile): new unity framework hybrid agent

### PR DESCRIPTION
**Pull request by ndesai-newrelic**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+</Step>
+
+<Step>
+### (iOS-only) Please ensure that your External Dependency Manager settings match the following configuration. 
+
+In your Unity IDE, navigate to Assets → External Dependency Manager → iOS Resolver → Settings: [#configure-ios]
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx:147]: @ndesai-newrelic What does a user have to look for under settings? Or what fields do they have to configure? Just having the screenshot is not enough I don't think

Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx]: basically customers need to follow this pod configurations to resolve ios dependency.
Screenshot 2023-12-13 at 11 15 38 AM

Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx]: customer need to add podfile to the main target always.

**Comment by ally-sassman** [comment]
@ndesai-newrelic Did you mean to close this PR?

**Pull request closed without merging**